lsp-mode.el

```diff
@@ -664,6 +664,13 @@ If this is set to nil, `eldoc' will show only the symbol information."
   :type 'boolean
   :group 'lsp-mode)
 
+(defcustom lsp-text-edit-applied-hook nil
```

Comment:
We have `lsp-before-apply-edits-hook`, shouldn't this be `lsp-after-apply-edits-hook`?